fpdf/fpdf.py

```diff
@@ -2059,32 +2088,43 @@ def _render_styled_cell_text(
         """
         Prints a cell (rectangular area) with optional borders, background color and
         character string. The upper-left corner of the cell corresponds to the current
-        position. The text can be aligned or centered. After the call, the current
-        position moves to the right or to the next line. It is possible to put a link
-        on the text.
+        position. The text can be aligned, centered or justified. After the call, the
+        current position moves to the requested new position. It is possible to put a
+        link on the text.
 
         If automatic page breaking is enabled and the cell goes beyond the limit, a
         page break is performed before outputting.
 
         Args:
+            text_line (TextLine instance): Contains the (possibly empty) tuple of
+                fragments to render.
             w (int): Cell width. Default value: None, meaning to fit text width.
                 If 0, the cell extends up to the right margin.
             h (int): Cell height. Default value: None, meaning an height equal
                 to the current font size.
-            styled_txt_frags (tuple): Tuple of fragments to render.
-                Default value: empty tuple.
             border: Indicates if borders must be drawn around the cell.
                 The value can be either a number (`0`: no border ; `1`: frame)
                 or a string containing some or all of the following characters
                 (in any order):
                 `L`: left ; `T`: top ; `R`: right ; `B`: bottom. Default value: 0.
-            ln (int): Indicates where the current position should go after the call.
-                Possible values are: `0`: to the right ; `1`: to the beginning of the
-                next line ; `2`: below. Putting 1 is equivalent to putting 0 and calling
-                `ln` just after. Default value: 0.
-            align (str): Allows to center or align the text inside the cell.
-                Possible values are: `L` or empty string: left align (default value) ;
-                `C`: center ; `R`: right align
+            newpos_x (Enum X): New current position in x after the call.
+                X.LEFT    - left end of the cell
```

Comment:
Shouldn't those details about all possible values be moved into docstrings in the enums definitions?

Reply:
Since the Enums are meant for (eventual) public consumption...
Yeah, sounds like a good idea.